webapp/templates/jury/submission.html.twig

```diff
@@ -742,7 +743,7 @@
                         <span style="float:right; border: 3px solid #438ec3; margin: 5px; padding: 5px;">
                             {% set imgUrl = path('jury_problem_testcase_fetch', {'probId': submission.problem.probid, 'rank': run.rank, 'type': 'image'}) %}
                             <a href="{{ imgUrl }}">
-                                <img src="data:image/{{ run.imageType }};base64,{{ runsOutput[runIdx].image_thumb | base64 }}"/>
+                                <img src="data:{{ EXTENSION_TO_MIMETYPE[run.imageType] }};base64,{{ runsOutput[runIdx].image_thumb | base64 }}"/>
```

Comment:
Does it not work with the variable in between? I think you can just use PHP here.